seqr/views/utils/orm_to_json_utils.py

```diff
@@ -855,4 +855,4 @@ def get_json_for_matchmaker_submissions(models, individual_guid=None, additional
 
 
 def get_json_for_matchmaker_submission(submission, **kwargs):
-    return _get_json_for_model(submission, get_json_for_models=get_json_for_matchmaker_submissions, **kwargs)
\ No newline at end of file
+    return _get_json_for_model(submission, get_json_for_models=get_json_for_matchmaker_submissions, **kwargs)
```

Comment:
your code editor seems to be removing newlines from the ends of files, but our code style is to have those new lines so please undo those changes

Reply:
this blocks approval